modules/system/assets/js/framework.js

```diff
@@ -99,10 +99,48 @@ if (window.jQuery.request !== undefined) {
 
                 delete data[inputName]
             }
+            
+            /*
+             * JavaScript Object to FormData, with support for nested objects, arrays and File objects.
+             * https://gist.github.com/ghinda/8442a57f22099bdb2e34
+             */
+            var objectToFormData = function(obj, form, namespace) {
+
```

Comment:
please remove the extra line from the start of the block, applies to your other blocks as well